projects/lights_control/src/lights_can_config.c

```diff
@@ -0,0 +1,80 @@
+#include "lights_can_config.h"
+#include "can_msg_defs.h"
+#include "lights_can.h"
+
+const LightsCanSettings s_settings_rear = {
+  // clang-format off
+  .event_lookup = {
```

Comment:
Please make sure this handles our actual CAN message format (see codegen-tooling)